tests/test_wsi_registration.py

```diff
@@ -0,0 +1,55 @@
+import urllib
+
+import numpy as np
+import PIL.Image as Image
+import pytest
+
+from tiatoolbox.tools.registration.wsi_registration import DFBRegistrtation
+
+
+def test_extract_features():
+    """Test for CNN based feature extraction function."""
+    main_url = "https://tiatoolbox.dcs.warwick.ac.uk/testdata/registration/"
+    urllib.request.urlretrieve(main_url + "HE_1_level8_gray.png", "sample.png")
+    fixed_img = np.asarray(Image.open("sample.png"))
+    urllib.request.urlretrieve(main_url + "HE_2_level8_gray.png", "sample.png")
```

Comment:
Please use remote_sample in `conftest.py`.

Reply:
Added to remote_samples files:
 https://github.com/TissueImageAnalytics/tiatoolbox/blob/e89c666bf952ab1f0854cf1c5e7679e48b614e2d/tiatoolbox/data/remote_samples.yaml#L112-L115